build-system/babel-plugins/babel-plugin-amp-mode-transformer/index.js

```diff
@@ -61,10 +73,24 @@ module.exports = function ({types: t}) {
           } else if (property.name === 'minified') {
             path.replaceWith(t.booleanLiteral(true));
           } else if (property.name === 'version') {
-            path.replaceWith(t.stringLiteral(INTERNAL_RUNTIME_VERSION));
+            path.replaceWith(t.stringLiteral(version));
           }
         }
       },
+      CallExpression(path) {
+        if (!modeNamespaceFound) {
+          return;
+        }
+
+        const {INTERNAL_RUNTIME_VERSION: version} = this.opts;
+        const {callee} = path.node;
+        if (
```

Comment:
Really, these should be updated to use `referencesImport`: https://astexplorer.net/#/gist/027ec54f10db0c8b67f30271097ff792/45deb3c152ce246ee14f43c11c9a698b4148d295

Reply:
Does that work for `getMode` too? It uses relative imports

Reply:
You'll need to resolve the relative path for each file, but otherwise yes.